marlowe-playground-client/src/Examples/JS/Contracts.purs

```diff
@@ -0,0 +1,166 @@
+module Examples.JS.Contracts where
+
+escrow :: String
+escrow =
+  """/* Parties */
+var alice = role("alice");
+var bob = role("bob");
+var carol = role("carol");
+
+/* Accounts */
+var alicesAccount = accountId(bigInt(0), alice);
+
+/* Value under escrow */
+var price = constant(bigInt(450));
+
+/* helper function to build Actions */
+
+var choiceName = "choice";
+
+var choiceIdBy = function (party) {
+                     return choiceId(choiceName, party);
+                 }
+
+var choiceBy = function(party, bounds) {
+                    return choice(choiceIdBy(party), bounds);
+                };
+
+
+var choiceValueBy = function(party) {
+                        return choiceValue(choiceIdBy(party));
```

Comment:
I'm adding this comment here but it's actually about the intellisense on the editor. When you hover over `choiceValue` it says it accepts type `any`. It seems that if you write the API in TypeScript you would get great intellisense.

Reply:
Oh weird, I see we are using TS and yet the intellisense is not showing the types for some reason